airbyte-commons-worker/src/main/java/io/airbyte/workers/config/ApiClientBeanFactory.java

```diff
@@ -67,6 +68,11 @@ public SourceApi sourceApi(final ApiClient apiClient) {
     return new SourceApi(apiClient);
   }
 
+  @Singleton
+  public DestinationApi destinationApi(final ApiClient apiClient) {
+    return new DestinationApi(apiClient);
+  }
```

Comment:
Added so we have the singleton available for injection directly - followed what was already being done for `SourceApi`